openshift/release/release-notes.md

```diff
@@ -0,0 +1,61 @@
+# Tektoncd cli TP-1.2
+
+# Tekton Cli v0.13.0
+
+## Features :sparkles:
+* Enable auto-select support in ClusterTask/TaskRun if only one is present https://github.com/tektoncd/cli/pull/1154
+* Use --prefix-name option for tkn clustertask start https://github.com/tektoncd/cli/pull/1179
+* Bump pipeline and triggers dep https://github.com/tektoncd/cli/pull/1191
+* Enable auto-select support in pipelineDescribe if only one pipeline is present https://github.com/tektoncd/cli/pull/1187
+* Enable auto select support in PipelineRunDescribe if only one PipelineRun Present https://github.com/tektoncd/cli/pull/1190
+
+## Fixes :bug:
+* Modify tkn version to accept namespace as ldflag and flag https://github.com/tektoncd/cli/pull/1092
+* Cancelling a pipelinerun which failed with Failed(Cancelled) throws an error https://github.com/tektoncd/cli/pull/1185
+* Fix deployment fetch issue for multiple namespaces https://github.com/tektoncd/cli/pull/1186
+
+## Misc :hammer:
+* Update Condition Tests from Test Builders to Structs https://github.com/tektoncd/cli/pull/1167
+* Update README Versions for v0.12.1 https://github.com/tektoncd/cli/pull/1173
+* Artwork/Logo added to the top of the README. https://github.com/tektoncd/cli/pull/1176
+* Fix rpm build for release https://github.com/tektoncd/cli/pull/1172
+* Update ClusterTask Tests from Test Builders to Structs https://github.com/tektoncd/cli/pull/1168
+* tests: import pipeline internal builders https://github.com/tektoncd/cli/pull/1177
+* Update PipelineResource Tests from Test Builders to Structs https://github.com/tektoncd/cli/pull/1182
+* Remove release-note block indentation in PR template https://github.com/tektoncd/cli/pull/1183
```

Comment:
Can we remove complete Misc? because those are internal changes not needed for end user

Reply:
I am fine removing it yes

Reply:
Yup good with removing, nothing user specific